content/blog/thank-you-reviewers-2020.md

```diff
@@ -0,0 +1,170 @@
+[website](https://anirudha.dev/) 
+[Twitter](https://twitter.com/kranirudha).
+
+
+### Anmol Krishan Sachdeva
+    Anmol is an International Tech Speaker, a Distinguished Guest Lecturer, a Tech Panelist, and has represented India at several reputed International Hackathons. He is a Deep Learning Researcher and has around 8+ publications. Recently, Anmol helped organize EuroPython 2020 and GeoPython & Python Machine Learning Conference 2020, and both of them were a huge success. He has done MSc in Advanced Computing (ML, AI, Robotics, Cloud Computing, and Computational Neuroscience) from University of Bristol, UK, and currently works at OLX Group as an SRE. In past, he has spoken at renowned conferences and tech forums like PyCon, EuroPython, GeoPython, and got invited as a Chief Guest / Guest of Honour at various events. He likes innovating, keeping in touch with new technological trends, and mentoring people.
+[Twitter](https://www.twitter.com/greatdevaks)
+[Linkedin](https://www.linkedin.com/in/greatdevaks)
+
+
+### Archana Iyer
+    Archana is an AI Engineer at Continental Automotive Singapore. She is a graduate student, based out of Singapore. She worked as a Deep Learning Engineer at a Pharma IT Company for 1.5 years where I worked on deep learning and NLP Applications. Apart from that, she has a keen interest in edge computing and computer vision. She created a course, “Choosing the Right Hardware for the Edge” for the Intel AI on Edge for IoT Developers Nanodegree on Udacity in April 2020. In her free time, she works at Women Who Code as a Leadership fellow for the Python/Cloud Tracks.
+
+
+### Charmi Chokshi
+    Charmi is a Machine Learning Engineer and one of the youngest Google Developers Expert in ML. She loves helping developers & students looking forward to growing their careers in the field of Artificial Intelligence via tech talks, mentoring sessions, and blogs.
+[Twitter](https://twitter.com/CharmiChokshi)
+
+
+### Chetan Giridhar
+    Chetan is a Python enthusiast and open source evangelist. He has been a speaker at Python conferences and takes keen interest in platform engineering, distributed systems and real time cloud communications. In his spare time Chetan reads about the world history and loves helping startups.
+
+
+### Derrick Joseph
+     Derrick is former PyCon speaker and has been a Pythonista since 2015, he started his career as a Data Scientist and later moved to Engineering. He has experience working with ML, Systems and has implemented business processes automations as generic frameworks.
+His current area of exploration is CPython and is working with Linkedin's Production Infrastructure Engineering team.
+[Twitter](https://twitter.com/C0d3eswaran)
```

Comment:
```suggestion
     Derrick is former PyCon speaker and has been a Pythonista since 2015, he started his career as a Data Scientist and later moved to Engineering. He has experience working with ML, Systems and has implemented business processes automations as generic frameworks. His current area of exploration is CPython and is working with Linkedin's Production Infrastructure Engineering team.
[Twitter](https://twitter.com/C0d3eswaran)
```